shipyard-controller/internal/handler/debughandler.go

```diff
@@ -180,3 +181,40 @@ func (dh *DebugHandler) GetEventByID(c *gin.Context) {
 
 	c.JSON(http.StatusOK, event)
 }
+
+// GetBlockingSequences godoc
+// @Summary      Get all blocking sequences for specific sequence
+// @Description  Get all the sequences which block a specific sequence
+// @Tags         Sequence
+// @Param        project			  path      string                    			true "The name of the project"
+// @Param        shkeptncontext       path      string                    			true "The Context of the sequence"
+// @Param        stage                path     string                    			true "The Stage of the sequences"
+// @Success      200                  {object}  []models.SequenceExecution          "ok"
+// @Failure      404                  {object}  models.Error             			"not found"
+// @Failure      500                  {object}  models.Error              			"Internal error"
+// @Router       /sequence/project/{project}/shkeptncontext/{shkeptncontext}/stage/{stage}/blocking [get]
```

Comment:
I would switch these two since state relates to project more than a keptn context. 
```suggestion
// @Router       /sequence/project/{project}/stage/{stage}/shkeptncontext/{shkeptncontext}/blocking [get]
```

Reply:
There will be a followup restructuring all the endpoints.